src/mainMenuScene.js

```diff
@@ -0,0 +1,36 @@
+import { ArcRotateCamera } from "@babylonjs/core/Cameras/arcRotateCamera";
+import { Color4 } from "@babylonjs/core/Maths/math";
+import { Scene, Vector3, Sound } from "@babylonjs/core";
```

Comment:
I know we discussed this, but this takes away all benefits of tree shaking. If it is intentional, leave it (of course :-) )

Reply:
this was more of an "let VSCode add them" and I'll get this cleaned up before merge - thing so no, not really intentional. 

I'm trying to think of what a simple script might do to check and/or fix these references, but that's for another time :smile: